src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Tizen/EmbeddingControls.tizen.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using Xamarin.Forms;
+using XLabel = Xamarin.Forms.Label;
+using XTextAlignment = Xamarin.Forms.TextAlignment;
+using System.Globalization;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class EmbeddingControls : ContentView
+	{
+		static readonly string PlayImagePath = "Xamarin.CommunityToolkit.Views.MediaElement.Tizen.Resources.img_button_play.png";
```

Comment:
One concern. I'd want to avoid adding these images to shared code (we don't want to increase nuget's size even when Tizen is not used).

CC @jfversluis @pictos @brminnick

Reply:
I thought it should be ok with 5.6KB images. Or I can consider drawing it using Shapes.

Reply:
I think you already update this with Shapes? :)